Connectors/Autocad/Speckle.Connectors.AutocadShared/Operations/Send/AutocadRootObjectBaseBuilder.cs

```diff
@@ -104,21 +125,53 @@ CancellationToken cancellationToken
           root.elements.Add(objectCollection);
         }
 
-        var result = ConvertAutocadEntity(entity, applicationId, objectCollection, instanceProxies, projectId);
+        var result = ConvertAutocadEntity(
+          entity,
+          applicationId,
+          objectCollection,
+          instanceProxies,
+          projectId,
+          referenceTransform // set this for top level instance proxies to use if needed
+        );
+
         results.Add(result);
 
         onOperationProgressed.Report(new("Converting", (double)++count / atomicObjects.Count));
       }
 
+      // 4 - Convert atomic definition objects
```

Comment:
@oguzhankoral this is the impact of two separate atomic object lists from the instance unpacker, looks bad imo